shwifty.cabal

```diff
@@ -41,12 +41,14 @@ library
     Shwifty.Pretty
     Shwifty.Types
   build-depends:
+    , aeson >= 1.5 && < 2
```

Comment:
The version bounds here are somewhat arbitrary.

Reply:
we should allow older versions, probably >= 1.0